tests/basic/tests.py

```diff
@@ -357,6 +357,23 @@ def test_hash(self):
             # hash)
             hash(Article())
 
+    def test_missing_hash_not_inherited(self):
+        class NoHash(models.Model):
+            def __eq__(self, other):
+                return super.__eq__(other)
```

Comment:
Maybe this should be just `pass` as `super.__eq__` shouldn't work (`super().__eq__` would). This is of course minor, as the line is never executed.